Parser.ipynb

```diff
@@ -407,14 +409,103 @@
     "display(csd_final)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = csd_final\n",
+    "ects = pd.read_csv(\"subjects_ects.csv\")\n",
+    "ects_dict = ects.set_index('Subjects')['Ects'].to_dict()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_courses = data.iloc[:,10:]\n",
+    "ects_total = 0\n",
+    "total = 0\n",
+    "gpa = 0\n",
+    "number = 0\n",
+    "gpa_list = []\n",
+    "ects_list = []\n",
+    "ects_needed = []\n",
+    "number_of_prior_courses = []\n",
+    "for i in range (0, 43):\n",
+    "    for column in data_courses.columns[:]:\n",
+    "        temp = data_courses.at[i,column]\n",
+    "        if temp >= 5:\n",
+    "            number = number + 1\n",
+    "            total = total + temp * ects_dict.get(column)\n",
+    "            ects_total = ects_total + ects_dict.get(column)\n",
+    "    #print('GPA: ', total / ects_total , 'ECTS: ', ects_total)\n",
+    "    number_of_prior_courses.append(number)\n",
+    "    ects_list.append(ects_total)\n",
+    "    ects_needed.append(240-ects_total)\n",
+    "    gpa_list.append(total / ects_total)\n",
+    "    ects_total = 0\n",
+    "    total = 0\n",
+    "    number = 0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Rename Columns ###"
+   ],
+   "execution_count": null,
+   "outputs": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change Courses Names\n",
+    "import json\n",
+    "from pandas.io.json import json_normalize\n",
+    "\n",
+    "with open(\"courses_ids_600000014.json\",\"r\") as json_file:\n",
+    "     json_file = json.load(json_file)\n",
+    "coded_courses = json_normalize(json_file['courses'])\n",
```

Comment:
Running Rename Columns block has exit this warning:
FutureWarning: pandas.io.json.json_normalize is deprecated, use pandas.json_normalize instead
```suggestion
    "coded_courses = pd.json_normalize(json_file['courses'])\n",
```